### ISSUE TITLE
Ephemeral does not remember dark preference

### ISSUE THREAD

**Issue by janxkoci** [cassidyjames/ephemeral#265]
After the recent update to `7.0.4`, the browser does not remember the setting for dark preference - new window is always created with this setting off. Previous versions remembered my last choice.

All other settings are remembered, such as zoom level or default search engin.

**Comment by cassidyjames** [comment]
Yep, this was a (knowing) regression to get the support for elementary OS 6 dark style shipping; it defaults to the system style and then allows a temporary override. I can look into bringing back a permanent override that interoperates with the dark style preference, but it gets more complex pretty quickly.

**Comment by janxkoci** [comment]
That makes sense. However my system (elementary OS 5) doesn't have any preference for dark style. Could it be detected? Maybe the permanent override could kick in only if the system doesn't send any signal for dark/light style preference?

**Comment by cassidyjames** [comment]
@janxkoci yep, exactly; that will work on elementary OS 5 and anything other than elementary OS 6.
